copyright

```diff
@@ -439,6 +439,16 @@ Copyright: tehhowch (https://github.com/tehhowch)
 License: CC-BY-SA-4.0
 Comment: Derived from works by Michael Zahniser (under the same license).
 
+Files:
+ images/effect/explosion/pug huge.png
+ images/effect/explosion/pug large.png
+ images/effect/explosion/pug medium.png
+ images/effect/explosion/pug small.png
+ images/effect/explosion/pug tiny.png
```

Comment:
These can all be condensed to `images/effect/explosion/pug*`, and if creating new explosions for different factions becomes the norm, you might want to put these explosions in their own folder similar to how the engine flares are divided, meaning you'd drop `pug` from the name of all the images and change this to `images/effect/explosion/pug/*`.

Reply:
You need the asterisk to represent how there are multiple files named “pug” in the explosion folder that you have the copyright for.